src/data/api-lapis.ts

```diff
@@ -116,7 +116,15 @@ export async function fetchMutationProportions(
   sequenceType: SequenceType,
   signal?: AbortSignal
 ): Promise<MutationProportionEntry[]> {
-  const url = await getLinkTo(`${sequenceType}-mutations`, selector, undefined, undefined, undefined, true);
+  const url = await getLinkTo(
+    `${sequenceType}-mutations`,
+    selector,
+    undefined,
+    undefined,
+    undefined,
+    true,
+    '0.01'
```

Comment:
The minimal selectable proportion is 0.001.

Reply:
I changed this to `0.001`